src/canvas-view/canvas/canvas-component.js

```diff
@@ -0,0 +1,406 @@
+(function () {
+  'use strict';
+
+  angular.module('patternfly.canvas')
+    .filter('trustAsResourceUrl', ['$sce', function ($sce) {
+      return function (val) {
+        return $sce.trustAsResourceUrl(val);
+      };
+    }])
+
+    //
+    // Component that generates the rendered chart from the data model.
+    //
+    .component('pfCanvas', {
+      templateUrl: "canvas-view/canvas/canvas.html",
+      bindings: {
+        chartDataModel: "=",
+        chartViewModel: "=?",
+        readOnly: "=?",
+        hideConnectors: "=?"
+      },
```

Comment:
We seem to flip back and forth between double quotes and single without a clear reason? Do you have a preference, or you up for making them consistent? If so encourage you to go back through and check them.

Reply:
I'm fine with double quotes. I can go back and make them consistent.